app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.player import Player
+from typing import List
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    return sum(rating.get_rating() for rating in team)
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
The type annotation for the `elves` parameter should use `List[Player]` instead of `list` for consistency and to adhere to best practices. Ensure you import `List` from the `typing` module.